src/Components/Classes/index.jsx

```diff
@@ -0,0 +1,174 @@
+import styles from './classes.module.css';
+import Table from './Table/table.jsx';
+import ClassForm from './Fprm/classFormEdit';
+import { useEffect, useState } from 'react';
+import ClassFormCreate from './Fprm/classFormCreate';
+
+function Projects() {
```

Comment:
The name of the component could be more representative, wdyt?

Reply:
Fixed